.github/workflows/build.yaml

```diff
@@ -46,10 +61,45 @@ jobs:
         PYTHON: python
     - name: Install dependencies
       run: make setup
+    - name: Create changelog and bump
+      if: ${{ github.ref == 'refs/heads/main' }}
+      run: |
+        git config --global user.name "Action User"
+        git config --global user.email "action@github.com"
+        git config --list --global
+        cz changelog
+        cz bump
     - name: Build the package
```

Comment:
Shouldn’t we check, test, build the package _before_ we bump?

Reply:
You know the answer ;-)

Reply:
I do 🤦🏻‍♂️